lib/fastlane/plugin/wpmreleasetoolkit/actions/common/check_for_toolkit_updates_action.rb

```diff
@@ -0,0 +1,77 @@
+require 'fastlane/action'
+require 'rubygems/command_manager'
+
+module Fastlane
+  module Actions
+    class CheckForToolkitUpdatesAction < Action
+      TOOLKIT_SPEC_NAME = 'fastlane-plugin-wpmreleasetoolkit'.freeze
+
+      def self.run(params)
+        updater = Gem::CommandManager.instance[:update]
+        installed_gems = updater.highest_installed_gems.select { |spec| spec == TOOLKIT_SPEC_NAME }
```

Comment:
Note that when run in the context of `bundle exec`, this will already rightfully use the rubygems environment provided by Bundler and specific to the current app's repo. Especially it will check for the gems in `vendor/bundler/…` (the location provided by `bundle exec gem env gemdir`, so the gem dir provided by Bundler's context, not the system-wide nor rbenv's ruby-version-wide one), which is exactly what we want.